### DISCUSSION TITLE
external load balancing of reads

### DISCUSSION THREAD

**Discussion by ajrpayne** [postgresml/pgcat#583] [Q&A]
We use a hardware load balancer that sits in front of our read servers, so pgcat would only see a single ip address(This we cannot change). If a single read server fails a health check then would all remaining read servers be banned due to having a shared ip? Can the ban time be disabled by setting it to 0?

**Comment by levkk** [comment]
Your hardware load balancer is blocking pgcat from doing read replica healthchecks and routing traffic to load balance reads. Unfortunately, this is not a setup that's compatible with pgcat's design. You'll need to use pgcat as your load balancer to take advantage of its features.

You could theoretically change the source code and disable banning of replicas, but I don't see a benefit from you using pgcat in this setup, unless you're using its other features like splitting reads and writes.

**Reply by ajrpayne** [reply]
We are looking at pgcat for it's splitting of reads and writes.

**Reply by levkk** [reply]
I think we can make the failover (banning of replicas) feature optional, I can imagine it to be useful to sometimes disable it.
